ivy/functional/frontends/paddle/tensor/manipulation.py

```diff
@@ -44,6 +44,33 @@ def cast(x, dtype):
     return ivy.astype(x, dtype)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("bool", "int32", "int64", "float16", "float32", "float64")},
+    "paddle",
+)
+@to_ivy_arrays_and_back
+def chunk(x, chunks, axis=0, name=None):
+    if ivy.shape(x) == ():
+        return [x]
+    else:
```

Comment:
No need for the `else` statement here as we already have a `return` above this line.

Reply:
Resolved @bipinKrishnan